tests/test_transform.py

```diff
@@ -120,33 +120,35 @@ def test_nest_fields():
         [{"a": "group_3", "b": "1", "c": "1"}, {"a": "group_3", "b": "1", "c": "1"}],
     ]
 
-    nested_df = transform.nest_fields(
+    nested_df = utils.nest_fields(
         df=df, grouping="a", new_column="e", drop_columns=["d"]
     )
     assert list(nested_df["e"]) == expected_column_e
 
 
-class TestCountGroupedTotal():
+class TestCountGroupedTotal:
     df = pd.DataFrame(
         {
             "col_1": ["a", "a", "a", "b", "c", "c", "c"],  # 3 'Ensembl IDs'
             "col_2": ["x", "y", "z", "x", "y", "z", "z"],  # 3 'biodomains'
             "col_3": ["1", "1", "2", "3", "2", "1", "3"],  # 3 'go_terms'
-            "col_4": ["m", "m", "n", "n", "o", "o", "o"]   # An extra column that should get ignored
+            "col_4": [
```

Comment:
Why is this one randomly split into one entry per line?

Reply:
Interesting. Looks like my auto-formatter went a little rogue. Thanks for noticing that!